server/src/uuid/model/entity/mod.rs

```diff
@@ -585,6 +586,28 @@ impl Entity {
     {
         let mut transaction = executor.begin().await?;
 
+        if payload.entity_type == EntityType::Solution
+            && sqlx::query!(
+                "SELECT uuid.id 
+                 FROM entity_link JOIN uuid ON uuid.id = entity_link.id 
```

Comment:
```suggestion
                 FROM entity_link JOIN uuid ON uuid.id = entity_link.child_id
```

Reply:
I think something went wrong when pulling, so that the original `entity_link.id` is back

Reply:
Yes thanks, I must have accidentally overwritten the change with the text editor.